sysmonconfig-export-block.xml

```diff
@@ -1322,6 +1323,7 @@
 			<!-- Executables dropped to suspicious folders -->
 			<TargetFilename condition="begin with">C:\Users\Public\</TargetFilename> <!-- often used staging directories; could cause false positives --> 
 			<TargetFilename condition="begin with">C:\Perflogs\</TargetFilename> <!-- often used staging directories --> 
+			<TargetFilename condition="begin with">C:\AppData\LocalLow\</TargetFilename> <!-- often used staging directories --> 
```

Comment:
This looks wrong

Reply:
You're right It's 100% wrong. Should be fixed now